lib/mods_display/fields/name.rb

```diff
@@ -146,9 +151,7 @@ def initialize(data)
       end
 
       def to_s
-        text = @name.dup
-        text << " (#{@roles.join(', ')})" if @roles
-        text
+        @name.dup
```

Comment:
I think this can just be `@name`.  We were calling `dup` (I think) because we were modifying the string and there was some weirdness going on otherwise, but now that we're not doing anything I think `dup` isn't necssary.

Reply:
Ok, good catch